### ISSUE TITLE
last drop out may affect the validation result

### ISSUE THREAD

**Issue by jalola** [wenwenyu/PICK-pytorch#77]
Since the last drop out of the encoder is not created as an attribute of the encoder module, the validation process will run it as well with drop out prob 0.1

**Comment by wenwenyu** [comment]
Hi, thanks for your reminder. We already pass `training` args to the last dropout function of the encoder, so it actually didn't affect the validation result. But to be consistent with my usage of other dropout, I have modified the call method here. Thanks again.
